### ISSUE TITLE
Keybase install of deleted phone app and verifying the device

### ISSUE THREAD

**Issue by DavidRaw** [keybase/keybase-issues#3554]
I logged in to keybase on my PC and can see I have 2 devices. Home PC and Phone. Also have various keys. Needed to reinstall keybase app on phone as had deleted it.

On phone I choose log in, enter my user name. I then get a message 'For security reasons, you need to authorise with an existing device. Which of your existing devices would you like to choose'

I have option of Phone or Home Computer. As I am on my phone I choose home computer. It then asks I set a public name for this phone: pick a device name. I pick phone 2.

I then get a screen saying 'On home computer, go to Devices Add Device New Phone.

Here I am stuck ? I proceeded by trying to connect my phone to my PC via bluetooth? but no luck. I then tried my laptop instead and logged in to key base repeated whole process again and linked my phone to Laptop via bluetooth but again nothing on the PC happens I have a barcode on phone or a secret phrase option but nothing going on on the PC or Laptop.

Can anyone help?
Thanks

**Comment by wildanmg** [comment]
Emmm i am too . My phone is lost . And when i want login keybase , i can't because my phone is lost

**Comment by Lucky225** [comment]
> I logged in to keybase on my PC and can see I have 2 devices. Home PC and Phone. Also have various keys. Needed to reinstall keybase app on phone as had deleted it.
> 
> On phone I choose log in, enter my user name. I then get a message 'For security reasons, you need to authorise with an existing device. Which of your existing devices would you like to choose'
> 
> I have option of Phone or Home Computer. As I am on my phone I choose home computer. It then asks I set a public name for this phone: pick a device name. I pick phone 2.
> 
> I then get a screen saying 'On home computer, go to Devices Add Device New Phone.
> 
> Here I am stuck ? I proceeded by trying to connect my phone to my PC via bluetooth? but no luck. I then tried my laptop instead and logged in to key base repeated whole process again and linked my phone to Laptop via bluetooth but again nothing on the PC happens I have a barcode on phone or a secret phrase option but nothing going on on the PC or Laptop.
> 
> Can anyone help?
> Thanks

On your PC that has keybase you need to got  to Devices > Add Device FROM THE KEYBASE APPLICATION - *not* from your OS. There's no bluetooth involved here it will present a QR code for you to scan from your phone.

**Comment by DavidRaw** [comment]
Thanks so much for coming back to me. I do not see the functionality with keybase to add a device. Perhaps the issue is that I deleted keybase off my PC and I log in through keybase.io? I can see I have 2 devices in account and various keys.I downloaded the software but it asked me to confirm with another device and then I am stuck?Thank youSent from my Samsung Galaxy smartphone.
-------- Original message --------From: Lucky225 <notifications@github.com> Date: 19/09/2019  09:37  (GMT+00:00) To: keybase/keybase-issues <keybase-issues@noreply.github.com> Cc: DavidRaw <davidjraw@gmail.com>, Author <author@noreply.github.com> Subject: Re: [keybase/keybase-issues] Keybase install of deleted phone app and verifying the device (#3554) 
I logged in to keybase on my PC and can see I have 2 devices. Home PC and Phone. Also have various keys. Needed to reinstall keybase app on phone as had deleted it.
On phone I choose log in, enter my user name. I then get a message 'For security reasons, you need to authorise with an existing device. Which of your existing devices would you like to choose'
I have option of Phone or Home Computer. As I am on my phone I choose home computer. It then asks I set a public name for this phone: pick a device name. I pick phone 2.
I then get a screen saying 'On home computer, go to Devices Add Device New Phone.
Here I am stuck ? I proceeded by trying to connect my phone to my PC via bluetooth? but no luck. I then tried my laptop instead and logged in to key base repeated whole process again and linked my phone to Laptop via bluetooth but again nothing on the PC happens I have a barcode on phone or a secret phrase option but nothing going on on the PC or Laptop.
Can anyone help?
Thanks

On your PC that has keybase you need to got  to Devices > Add Device FROM THE KEYBASE APPLICATION - not from your OS. There's no bluetooth involved here it will present a QR code for you to scan from your phone.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/keybase/keybase-issues/issues/3554?email_source=notifications\u0026email_token=AHOKRSTTBFVCGPWO25QV2W3QKM23TA5CNFSM4IYA24GKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD7CWCHQ#issuecomment-533029150",
"url": "https://github.com/keybase/keybase-issues/issues/3554?email_source=notifications\u0026email_token=AHOKRSTTBFVCGPWO25QV2W3QKM23TA5CNFSM4IYA24GKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD7CWCHQ#issuecomment-533029150",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by plttn** [comment]
@DavidRaw At this point, you'll have to do this

https://keybase.io/_/reset_queue

You need an actual client app to activate a new device, not the website.

**Comment by Lucky225** [comment]
> @DavidRaw At this point, you'll have to do this
> 
> https://keybase.io/_/reset_queue
> 
> You need an actual client app to activate a new device, not the website.

Unless he has a paper key, correct.  Keybase requires each new device to be SIGNED by an existing device, if you uninstalled from both your PC and the phone you've lost your private keys which is what proves you're you.  That's why paper keys are essential in the case of device being lost or compromised, a paper key is the only way back in to your account to prevent keybase from acting like a man-in-the-middle that would be able to spy on you without your keys

**Comment by plttn** [comment]
It shows paper keys under devices so I made the assumption that he does not have any paper keys.

**Comment by DavidRaw** [comment]
Yes I can see keys under devices I will look at the reset link you sent maybe that enables me to input the keys?Thank you for helping.Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: Jack Platten <notifications@github.com> Date: 20/09/2019  02:49  (GMT+00:00) To: keybase/keybase-issues <keybase-issues@noreply.github.com> Cc: DavidRaw <davidjraw@gmail.com>, Mention <mention@noreply.github.com> Subject: Re: [keybase/keybase-issues] Keybase install of deleted phone app and verifying the device (#3554) It shows paper keys under devices so I made the assumption that he does not have any paper keys.

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/keybase/keybase-issues/issues/3554?email_source=notifications\u0026email_token=AHOKRSVABST4WWWS34IRT43QKQT4NA5CNFSM4IYA24GKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD7FJO7Y#issuecomment-533370751",
"url": "https://github.com/keybase/keybase-issues/issues/3554?email_source=notifications\u0026email_token=AHOKRSVABST4WWWS34IRT43QKQT4NA5CNFSM4IYA24GKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD7FJO7Y#issuecomment-533370751",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
